doc/classes/RenderingServer.xml

```diff
@@ -1450,6 +1450,7 @@
 			<description>
 				Returns the name of the video adapter (e.g. "GeForce GTX 1080/PCIe/SSE2").
 				[b]Note:[/b] When running a headless or server binary, this function returns an empty string.
+				[b]Note:[/b] On the web platform, some browsers such as Firefox may report a different, fixed GPU name such as "GeForce GTX 980" (regardless of the user's actual GPU model). This is done to make fingerprinting more difficult.
```

Comment:
We'll need to unify the casing of "Web" vs "web", and define some guidelines.
```
$ rg -g"*.xml" " Web[, ]" | wc -l
45
$ rg -g"*.xml" " web[, ]" | wc -l
30
```

I think at least when referring to Godot's "Web" platform, it should be capitalized, like it is in `OS.get_name()`.

Then maybe when referring to "the web", it can be lowercase.

Here with "the web platform", are we talking about the web as a platform, or Godot's "Web" platform?

Reply:
> Here with "the web platform", are we talking about the web as a platform, or Godot's "Web" platform?

I'd say the latter, but I don't have a strong opinion on it.

Reply:
I guess this can/should be handled in a separate PR anyway, so let's merge this one.